### PR TITLE
Tools first implementation

### PR DESCRIPTION
**Pull request by alexlemaire**
Opening a new pull request for the tools first implementation.

I also tested if the template as is worked and it seems like everything is fine: API deployed, lambda is working and everything is uploaded at the root of the bucket under the name defined via `service.name` in `serverless.json`.

### CODE REVIEW

File: tools/update-version/helpers/get-version.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const fs = require('fs')
+
+module.exports = () => {
+  const pjson = JSON.parse(fs.readFileSync('package.json', 'utf8'))
```
Comment (Holger-Will) [inline tools/update-version/helpers/get-version.js]: why not just `require("package.json")`?

**Comment by alexlemaire** [comment]
I realized that my `getVersion` helper wasn't very useful (kinda boils down to a one liner) so I moved the logic in the main file as a one line command instead

File: tools/update-version/helpers/get-version.js (hunk 1 above)
Comment (Holger-Will) [reply tools/update-version/helpers/get-version.js]: on the other hand, why not keep readFile in...

**Comment by Holger-Will** [comment]
looking very good, this can be merged!

**Comment by Holger-Will** [comment]
next time it might be better to have two PRs one per tools ;-)

**Comment by alexlemaire** [comment]
> next time it might be better to have two PRs one per tools ;-)

Alright will do that then next time. Merging now!

**Pull request merged**